tools/cli/lib/installcmd.c

```diff
@@ -218,16 +217,20 @@ TDNFCliAlterCommand(
 
     if(pSolvedPkgInfo->nNeedAction)
     {
+        int32_t opt = 0;
+
         if(!pCmdArgs->nAssumeYes && !pCmdArgs->nAssumeNo)
         {
-            printf("Is this ok [y/N]:");
-            if (scanf("%c", &chChoice) != 1)
+            printf("Is this ok [y/N]: ");
+
+            opt = getchar();
+            if (tolower(opt) != 'y' && tolower(opt) != 'n')
```

Comment:
shall we check for opt != EOF? As i am not sure the behaviour of tolower if we pass EOF as an input?

Reply:
EOF is CTRL-D so it's still an invalid input so we must fail. So no need to check again